code/game/machinery/newscaster.dm

```diff
@@ -165,6 +165,8 @@ var/list/obj/machinery/newscaster/allCasters = list() //Global list that will co
 	var/allow_comments = 1
 	luminosity = 0
 	anchored = 1
+	var/const/articlefont = "Verdana"
+	var/const/signfont = "Times New Roman"
```

Comment:
I just did the same as it was done in [Paper.dm](https://github.com/tgstation/-tg-station/blob/bff2e62a0f28e7838ad19f1b955e0f4250362e02/code/modules/paperwork/paper.dm).
I change it there too, but would like to do it in a fixing PR for after [both my PRs](https://github.com/tgstation/-tg-station/pull/7287) are merged.
Need to do as I [discussed with RemieRichards](https://github.com/tgstation/-tg-station/pull/7287#issuecomment-71113858).